pkg/cmd/experimental/buildchain/buildchain.go

```diff
@@ -94,12 +93,29 @@ func (o *BuildChainOptions) Complete(f *clientcmd.Factory, cmd *cobra.Command, a
 	}
 	o.c, o.t = oc, oc
 
-	// Parse user input
-	o.name, o.tag, err = buildChainInput(args[0])
-	if err != nil {
-		return cmdutil.UsageError(cmd, err.Error())
+	parts := strings.Split(args[0], "/")
```

Comment:
The resource builder will try to construct the istag, but if that doesn't exist *currently*, we will get an error. Instead, by using the RESTMapper to just resolve the syntax, we let the graph code to create graphs for tags that are outputs for builds (istags that aren't created yet).